lsp/server/src/server.ts

```diff
@@ -201,87 +194,122 @@ async function dockerContainer(folder: WorkspaceFolder) {
 		});
 		startedSA4U_Z3 = true;
 	} catch (e) {
-		console.log(e);
+		connection.sendNotification('ServerError', `Failed to start the docker container: ${e}`);
 	}
 }
 
-connection.onShutdown(() => {
+async function changedConfiguration(change:DidChangeConfigurationParams) {
```

Comment:
There should be a space following the `:`

Reply:
Fixed